configs/templates/skylab-dev/spack.yaml

```diff
@@ -28,8 +28,8 @@ spack:
     - bacio@2.4.1
     - bison@3.8.2
     - bufr@11.7.1
-    - crtm@2.3.0
-    - crtm@v2.3-jedi.4
+    - crtm@2.4.0
+    - crtm@v2.4_jedi
```

Comment:
This version of CRTM is going to work with UFO of skylab 2 release?

Reply:
That is my understanding, yes.

Reply:
Let's ask @BenjaminTJohnson to make sure that my understanding that we'll use those two versions is correct.

Reply:
Just confirmed by Francois V that this is the correct tag.